src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -52,11 +56,11 @@ var ChartsPanel = React.createClass({
 
     getInitialState() {
         return {
-            chartType : SCATTER,
-            optionsShown : true,
+            chartType: SCATTER,
+            optionsShown: true,
             widthPx: 700,
             heightPx: 300,
-            debouncedResize: debounce(this.onResize, 500, {'leading':false, 'trailing':true})};
+            debouncedResize: debounce(this.onResize, 200, {'leading':false, 'trailing':true, 'maxWait':10000})};
```

Comment:
`'leading':false, 'trailing':true` are the defaults so I don't think you need them